tests/FSharpLint.Core.Tests/Rules/Smells/InterpolatedStringWithNoSubstitution.fs

```diff
@@ -0,0 +1,57 @@
+﻿module FSharpLint.Core.Tests.Rules.Smells.InterpolatedStringWithNoSubstitution
+
+open NUnit.Framework
+open FSharpLint.Rules
+open FSharpLint.Core.Tests
+
+[<TestFixture>]
+type TestSmellsInterpolatedStringWithNoSubstitution() =
+    inherit TestAstNodeRuleBase.TestAstNodeRuleBase(InterpolatedStringWithNoSubstitution.rule)
+
+    [<Test>]
+    member this.``Interpolated string with no substitutions must generate error``() =
+        this.Parse
+            """let someString = "foo"
+Console.WriteLine $"someString"
+"""
+
+        Assert.IsTrue(this.ErrorsExist)
+        Assert.IsTrue(this.ErrorExistsAt(2, 18))
+
+    [<Test>]
+    member this.``Interpolated string with a substitutions must not generate error``() =
+        this.Parse
+            """let someString = "foo"
+Console.WriteLine $"{someString}"
+"""
+
+        this.AssertNoWarnings()
+
+    [<Test>]
+    member this.``Using sprintf with no substitutions must generate error``() =
+        this.Parse
+            """let someString = "foo"
+sprintf "someString"
+"""
+
+        Assert.IsTrue(this.ErrorsExist)
+        Assert.IsTrue(this.ErrorExistsAt(2, 0))
+
+    [<Test>]
+    member this.``Using sprintf with substitutions must not generate error``() =
+        this.Parse
+            """let someString = "foo"
+let bar = sprintf "someString = %s" someString
+let baz = someString |> sprintf "someString = %s"
```

Comment:
@webwarrior-ws let's change this code snippet to not generate an F# compilation warning